.babelrc

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
Note: We might need to upgrade some of the packages to make our metronome codebase be able to compile using Node 14, as it throws an error when you try compiling it with Node 14.